stock_picking_move_package_to_package/tests/test_move_package_to_package.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests import TransactionCase
+
+
+class TestStockPicking(TransactionCase):
+    def setUp(self):
```

Comment:
Use  setUpClass instead of the setUp in tests.
Disable tracking unless you need it for functional purposes

    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))